### PR TITLE
Syntax highlighting

### PR DESCRIPTION
**Pull request by john-zou**
image

Please make it into a better color, I have no design capacity

### CODE REVIEW

**Comment by Dara2004** [comment]
Can we use green instead of pink?! Looks great!!!

**Comment by john-zou** [comment]
image
Tada

File: src/App.css [hunk 1]
```diff
@@ -88,3 +88,9 @@ body {
   font-weight: bold !important;
   color: #666666 !important;
 }
+
+.syntax-highlight {
+  color: greenyellow;
```
Comment (Dara2004) [inline src/App.css:93]:
```suggestion
  color: #89c999;
```

Comment (Dara2004) [reply src/App.css:93]: how's this?

Comment (john-zou) [reply src/App.css]: I am color blind so it all looks grey to me

**Pull request merged**